sklearn/datasets/base.py

```diff
@@ -352,8 +352,9 @@ def load_iris(return_X_y=False):
         Dictionary-like object, the interesting attributes are:
         'data', the data to learn, 'target', the classification labels,
         'target_names', the meaning of the labels, 'feature_names', the
-        meaning of the features, and 'DESCR', the
-        full description of the dataset.
+        meaning of the features, 'DESCR', the
+        full description of the dataset, 'filename' (added in version 0.20),
```

Comment:
I am not in favor of  `(added in version 0.20)` in the narrative.

I would add a directive on the bottom of data

```python
        the physical location of iris csv dataset.

        .. versionadded:: 0.20
           The ``filename`` attribute.
```